### PR TITLE
trivia script

### PR DESCRIPTION
**Pull request by jboretsky**
trivia script which grabs questions/solutions from jservice.io, fully functional but want to add a database to score points. Questions are pretty hard.

@cpjk How do i rebase all of these into 1 commit?

### CODE REVIEW

**Comment by cpjk** [comment]
@jboretsky don't squash it into a single commit. Many small commits are way  better than a single large one.

Looks good though.

It would be nice if you could improve some of the commit messages RE: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

File: scripts/trivia.coffee [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      msg.reply "Incorrect!"
+
+  #command to turn off trivia
+  robot.respond /trivia off/, (msg) ->
+    if triviaOn
+      triviaStop(msg) 
+      msg.send "Trivia is now off"
+    else msg.send "Trivia is not in progress"
+      
+
```
Comment (cpjk) [inline scripts/trivia.coffee:66]: One too many newlines here.

File: scripts/trivia.coffee [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Description:
+#   start trivia round 
+#
+# Commands:
+#   woz trivia
+#   trivia help
+
+module.exports = (robot) ->
+
+  triviaOn = false;
```
Comment (andreirtaylor) [inline scripts/trivia.coffee]: You shouldnt use semicolons in Coffeescript

File: scripts/trivia.coffee [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    solveAttempt = msg.match[1]
+    if solveAttempt.toLowerCase() == currentSolution.toLowerCase()
+      msg.reply "Ding Ding Ding, you are correct!"
+      triviaStop(msg)
+      triviaStart(msg)
+    else
+      msg.reply "Incorrect!"
+
+  #command to turn off trivia
+  robot.respond /trivia off/, (msg) ->
```
Comment (andreirtaylor) [inline scripts/trivia.coffee]: You might as well make this case insensitive seeing as all the other one are.

**Comment by andreirtaylor** [comment]
Looks good to me.

**Comment by andreirtaylor** [comment]
I'm actually wondering how well this will work if people are playing in different room. Pulling in the changes is the only way I know how to test that.

**Comment by andreirtaylor** [comment]
As I suspected it is only possiible to have one instance of trivia running. If nobody is a jerk this will work, but it is possible to ruin someone else's game by running the "trivia stop" command. NOt sure how to get around that limitation

**Pull request merged**